File: server/src/main/java/pt/at/sme/InitialValues.java
package pt.at.sme;

public class InitialValues {

    public String getNif() {
        return nif;
    }

    public void setNif(String nif) {
        this.nif = nif;
    }

    public String nif;

    public InitialValues(String nif){
        this.nif = nif;
    }
}

File: common/src/commonMain/kotlin/pt/at/sme/kotlin/util/InitialValues.kt
package pt.at.sme.kotlin.util

import kotlinx.serialization.Serializable
import pt.at.sme.kotlin.priorNotification.PriorNotificationForm
import kotlin.js.JsExport

/**
 * Initial values passed to client-side ImportForm
 */
@Serializable
@JsExport
data class InitialValues(
    val nif: String
//    val mode: FormMode,
//    val preFilledForm: PriorNotificationForm?
)
